mathesar_ui/src/pages/schema/SchemaOverview.svelte

```diff
@@ -42,8 +49,29 @@
         {/if}
       </svelte:fragment>
     </OverviewHeader>
-    {#if isTablesLoading}
+    {#if tablesRequestStatus.state === 'processing'}
       <TableSkeleton numTables={schema.num_tables} />
+    {:else if tablesRequestStatus.state == 'failure'}
+      <ErrorBox>
+        <p>{tablesRequestStatus.errors[0]}</p>
+        <div>
+          <Button
+            on:click={() => {
+              if ($currentSchemaId) {
+                void refetchTablesForSchema($currentSchemaId);
```

Comment:
This component already has a `schema` property that's passed down. Do not utilize the currentSchemaId store here, use `schema.id` instead.